scripts/install.sh

```diff
@@ -23,6 +23,7 @@ function usage(){
     echo "    --issuerEndpoint (-i): This is the endpoint that mints the access_tokens and will also be the issuer in the access_token as well."
     echo "    --oAuth2ApiEndpoint (-o): this is probably similar to your issuer endpoint but is the prefix to all OAuth2 APIs"
     echo "    --patientPickerEndpoint (-p): SMART on FHIR supports launch contexts and that will typically include a patient picker application that will proxy the /token and /authorize requests."
+    echo "    --alarmSubscriptionEndpoint: The HTTPS endpoint to be configured as a subscriber on the CloudWatch Alarm SNS Topic."
     echo "    --lambdaLatencyThreshold: lambda latency threshold in ms (Default: 3000) "
     echo "    --apigatewayServerErrorThreshold: API gateway 5xxerror threshold (Default: 3)"
```

Comment:
Missing inclusion of `apigatewayMetricsEnabled` and `apigatewayLatencyThreshold` here.

Reply:
done

Reply:
I will open PR to smart-mainline branch as well when this changes is merged to feature/merged_forks as per discussion.

Reply:
Should we do the inverse? Merge to smart-mainline, then have feature/merge_forks pull in these changes from smart-mainline. Or, will it not matter if we merge this separately to both branches? I want to avoid any unnecessary conflicts when we are ready to merge feature/merge_forks to smart-mainline.

Reply:
okay sounds good. I will Open PR to smart-mainline.